run.js

```diff
@@ -51,7 +51,9 @@ function start () {
 	});
 
 	wdi.graphicDebug = new wdi.GraphicDebug({debugMode: false});
-	var app = new Application();
+	var app = new Application({
+		supportHighDPI: false
```

Comment:
I'm aware that initializing it to false is pointless but I'm adding this initialization as a template to show what config options it has.